src/features/transaction-wizard/transaction-wizard-page.test.tsx

```diff
@@ -0,0 +1,253 @@
+      const original = await vi.importActual<{ useWallet: () => ReturnType<typeof useWallet> }>('@txnlab/use-wallet')
+      vi.mocked(useWallet).mockImplementation(() => {
+        return {
+          ...original.useWallet(),
+          activeAddress: testAccount.addr,
+          signer: testAccount.signer,
+          status: 'active',
+          isActive: true,
+          isReady: true,
+        }
+      })
+    })
+
+    describe('and a payment transaction is being sent', () => {
+      it('reports validation errors when required fields have not been supplied', () => {
+        return executeComponentTest(
+          () => {
+            return render(<TransactionWizardPage />)
+          },
+          async (component, user) => {
+            const sendButton = await waitFor(() => {
+              const sendButton = component.getByRole('button', { name: sendButtonLabel })
+              expect(sendButton).not.toBeDisabled()
+              return sendButton!
+            })
+
+            await user.click(sendButton)
+
+            await waitFor(() => {
+              const requiredValidationMessages = component.getAllByText('Required')
+              expect(requiredValidationMessages.length).toBeGreaterThan(0)
+            })
+          }
+        )
+      })
+
+      it('succeeds when all fields have been correctly supplied', async () => {
+        const { testAccount } = localnet.context
+        const testAccount2 = await localnet.context.generateAccount({ initialFunds: algo(0) })
+
+        await executeComponentTest(
+          () => {
+            return render(<TransactionWizardPage />)
+          },
+          async (component, user) => {
+            const sendButton = await waitFor(() => {
+              const sendButton = component.getByRole('button', { name: sendButtonLabel })
+              expect(sendButton).not.toBeDisabled()
+              return sendButton!
+            })
+
+            const senderInput = await component.findByLabelText(/Sender address/)
+            fireEvent.input(senderInput, {
+              target: { value: testAccount.addr },
+            })
+
+            const receiverInput = await component.findByLabelText(/Receiver address/)
+            fireEvent.input(receiverInput, {
+              target: { value: testAccount2.addr },
+            })
+
+            const amountInput = await component.findByLabelText(/Amount/)
+            fireEvent.input(amountInput, {
+              target: { value: '0.5' },
+            })
+
+            await user.click(sendButton)
+
+            const transactionId = await waitFor(
+              () => {
+                expect(component.queryByText('Required')).not.toBeInTheDocument()
+                const transactionIdComponent = getByDescriptionTerm(component.container, transactionIdLabel)
+                expect(transactionIdComponent).toBeDefined()
+                return transactionIdComponent.textContent!
+              },
+              { timeout: 10_000 }
```

Comment:
I think this should be `5_000` because there is a max 5 seconds timeout for vitest test https://vitest.dev/api/#test.

Reply:
I upped it to 20 seconds in the vitest config as part of this PR.